README.md

```diff
@@ -1,2 +1,3 @@
 # Basic project setup
-## Hello World
\ No newline at end of file
+## Hello World
```

Comment:
There's a strange new "Hello World" line here.

Reply:
It's ok

Reply:
Well then you're good to go!

Reply:
Thank! Starting merge